shpc/client/help.py

```diff
@@ -242,3 +242,41 @@
   # Filter all modules to those with "python"
   $ shpc show --filter python
 """
+
+upgrade_description = """Upgrade software to the latest version.
+
+  # Upgrade a specific software to its latest version and give option to uninstall older versions or not.
+  # Do not include the version in the command
+  $ shpc upgrade quay.io/biocontainers/samtools
+
+  # Upgrade all software
+  $ shpc upgrade --all
+      OR
+  $ shpc upgrade -a    
+
+  # Show a list of the latest versions available for only the user's outdated software without upgrading them.
+  $ shpc upgrade --dry-run 
+      OR
+  $ shpc upgrade -d
```

Comment:
We discussed this before

Reply:
@muffato do you think this is necessary?

Reply:
I think that's been addressed already ?
```console
$ shpc upgrade
usage: shpc upgrade [-h] [--all] [--dry-run] [--force] [--module-sys {lmod,tcl}] [--container-tech {singularity,podman,docker}] [upgrade_recipe]
shpc upgrade: error: Incomplete command. The following arguements are required: upgrade_recipe, --all, or -h for more details 
```